File: URI/1 - INICIANTE/Python/1074 - ParOuImpar.py
x = int(input())
palavra = []
for y in range(0,x,1):
    num = int(input())
    if num != 0:
        if num % 2 == 0:
            palavra.append('EVEN')
        elif num % 2 != 0:
            palavra.append('ODD')
        if num > 0:
            palavra[y] += ' POSITIVE'
        else:
            palavra[y] += ' NEGATIVE'
    else:
        palavra.append('NULL')

for y in range(0, x, 1):
    print(palavra[y])

File: URI/1 - INICIANTE/C/1074 - ParOuImpar.c
#include <stdio.h>

int main()
{
	int num, x, y;
	
	scanf(" %d", &x);
	for(y = 0; y < x; y++)
	{
		scanf(" %d", &num);
		if(num != 0)
		{
			if (num % 2 == 0)
			{
				printf("EVEN ");
			}
			else{
				printf("ODD ");
			}
			if (num > 0){
				printf("POSITIVE\n");
			}
			else{
				printf("NEGATIVE\n");
			}
		}else{
			printf("NULL\n");
		}
	}
	
	return 0;
}
